app/src/org/commcare/utils/MediaUtil.java

```diff
@@ -257,7 +267,7 @@ private static Bitmap getBitmapScaledByTargetOrContainer(String imageFilepath,
         double widthScaleDownFactor =  (double)boundingWidth / originalWidth;
         // Choosing the larger of the scale down factors, so that the image still fills the entire
         // container
-        double dominantScaleDownFactor = Math.max(heightScaleDownFactor, heightScaleDownFactor);
+        double dominantScaleDownFactor = Math.max(widthScaleDownFactor, heightScaleDownFactor);

```

Comment:
@amstone326 `widthScaleDownFactor` wasn't being used, so I assume that you meant to use it here. Can you please verify that this is correct. If it isn't, please remove `widthScaleDownFactor`